requirements.txt

```diff
@@ -1,5 +1,5 @@
 coverage>=4.0.2
-Django>=1.11
+Django>=1.8
```

Comment:
I had initially thought that `model._meta.get_fields()` was only valid for v1.11, but later found that it exists on v1.8 as well. I have run the unit tests against all versions from 1.8 to 1.11 to confirm.